client/client_utils.cpp

```diff
@@ -0,0 +1,60 @@
+#include "client_utils.h"
+
+Operation genRandOperation() {
+	float r = (float) rand() / RAND_MAX;
+	int key = rand() % KEY_MAX;
+  
+	Operation op;
+	op.__set_op(r < 0.5 ? OpType::PUT : OpType::GET);
+  op.__set_key(std::to_string(key));
+
+	std::string value;
+
+  // If operation is GET, then set value to random bytes
+  // If operation is PUT, then set value to random int value
+	if (op.op == OpType::GET) {
+		char rand_val[VALUE_SIZE];
+		randombytes_buf(rand_val, VALUE_SIZE);
+		value = std::string(rand_val);
+	} else {
+		int put_val = rand() % VAL_MAX;
+		value = std::to_string(put_val);
+	}
+	op.__set_value(clientEncrypt(value));
+
+	return op;
+}
+
+Operation getSeedOperation(std::string& line) {
+  std::istringstream ss(line);
+  std::string operation, key, value;
+
+  std::getline(ss, operation, ',');
+  std::getline(ss, key, ',');
+  std::getline(ss, value, ',');
```

Comment:
I think this will work for our application (at least for now) but in a real CSV there can be strings that I think use quotation marks or something to delimit them, so you could have
`PUT,description,"this is a description with spaces but still one CSV column"`

Reply:
Yep that though occurred to me as well. We should definitely figure out how we want to go about CSV parsing (ie. use a library or implement ourselves)